lib/spotlight/dor/indexer.rb

```diff
@@ -61,27 +85,52 @@ def add_donor_tags sdb, solr_doc
       insert_field solr_doc, 'donor_tags', donor_tags, :symbol # this is a _ssim field
     end
 
+    # add the folder number to solr_doc as folder_ssi field (note: single valued!)
+    #   data in location/physicalLocation or in relatedItem/location/physicalLocation
+    # TODO:  push this up to stanford-mods gem?  or should it be hierarchical series/box/folder?
+    def add_folder(sdb, solr_doc)
+      # see spec for data from actual collections
+      #   _location.physicalLocation should find top level and relatedItem
+      folder_num = sdb.smods_rec._location.physicalLocation.map do |node|
+        val = node.text
+        match_data = val.match(/Folder:? ?(.+)/i)
+        next if match_data.blank?
+        result = match_data[1].rstrip
+        # Menuez collection may have folder followed by Sleeve then Frame
+        match2_data = result.match(/(.*),? ?Sleeve/i)
```

Comment:
uh ... i think you're talking about standard practice at spec for collections future AND past.  I think it's unlikely, but it's certainly worth a question.  I suspect it really depends on what sort of stuff it is.